RecoLocalCalo/HcalRecAlgos/interface/MahiFit.h

```diff
@@ -81,6 +81,42 @@ struct MahiNnlsWorkspace {
 
 };
 
+struct MahiDebugInfo {
+
+  int   nSamples;
+  int   soi;
+
+  bool  use3;
+
+  float inTimeConst;
+  float inDarkCurrent;
+  float inPedAvg;
+  float inGain;
+  
+  float inNoiseADC[10];
```

Comment:
if this and other 10 have a relationship to MaxSVSize or MaxPVSize, please use the constants instead of magic numbers.